lib/mhv_ac/registration_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'common/models/form'
+
+module MHVAC
+  class RegistrationForm < Common::Form
+    attribute :icn, String
```

Comment:
Going to spawn a ticket to work this out as it is beyond the scope of a PR comment thread.